gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -178,6 +181,42 @@ def table_exist(client, project_id, dataset_id, table_id):
   return True
 
 
+def table_empty(project_id, dataset_id, table_id):
+  client = bigquery.Client(project=project_id)
+  query = 'SELECT count(0) AS num_rows FROM {DATASET_ID}.{TABLE_ID}'.format(
+      DATASET_ID=dataset_id, TABLE_ID=table_id)
+  query_job = client.query(query)
+  num_retries = 0
+  while True:
+    try:
+      results = query_job.result(timeout=300)
+    except TimeoutError as e:
+      logging.warning('Time out waiting for query: %s', query)
+      if num_retries < BQ_NUM_RETRIES:
+        num_retries += 1
+        time.sleep(90)
+      else:
+        raise e
+    else:
+      if results.total_rows == 1:
+        break
+      else:
+        logging.error('Query did not returned expected # of rows: %s', query)
+        if num_retries < BQ_NUM_RETRIES:
+          num_retries += 1
+          time.sleep(90)
+        else:
+          raise ValueError(
+            'Expected 1 row in query result, got {}'.format(results.total_rows))
+
+  row = list(results)[0]
+  col_names = row.keys()
+  if set(col_names) != {'num_rows'}:
```

Comment:
Since output column is repeated in several places, lets put it into a separate const var as well.